addons/gdUnit4/src/core/parse/GdFunctionArgument.gd

```diff
@@ -20,7 +20,7 @@ func name() -> String:
 
 
 func default() -> Variant:
-	return convert(_default_value, _type)
+	return type_convert(_default_value, _type)
```

Comment:
Thanks for providing a fix, but this will break the compatibility to Godot 4.1.x.